### DISCUSSION TITLE
2022-01-10 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#171] [Core]
# TL;DR

Most people on vacation during last couple weeks. 

Stay tuned for the Q1 okrs next week.

# Done

@mzhangmzz - Add test for PR https://github.com/near/nearcore/pull/5747
@mzhangmzz - Debug testnet OOM issue
@mzhangmzz - Look into chunk only producers & testing for chunk producing and processing

@Longarithm - Script for computing rocksdb stats - col size, num entries, etc https://github.com/near/nearcore/pull/6005

@pmnoxx - Replace cached with lru for near-chain https://github.com/near/nearcore/pull/5871
@pmnoxx - Fixed tests will print output on error only https://github.com/near/nearcore/pull/6023
@pmnoxx - A lot of refactoring / code quality improvement PRs (30-40 prs)
@pmnoxx - Upgrade minimum nnetwork protocol version + remove old unused messages https://github.com/near/nearcore/pull/5611
@pmnoxx - Fix mac release script https://github.com/near/nearcore/pull/5952
@pmnoxx - Improvements to near-memory-tracker - 10 PRs https://github.com/near/near-memory-tracker

@mm-near - oncall and adding documentation on how to handle different alerts
